eos-updater/poll-common.c

```diff
@@ -514,8 +514,9 @@ should_follow_checkpoint (OstreeSysroot     *sysroot,
   g_autoptr(GHashTable) hw_descriptors = NULL;
   const gchar *sys_vendor, *product_name;
   gboolean is_eos3a_to_eos4 =
```

Comment:
Since we’re now allowing transitions which aren’t `eos3a` → `eos4`, and disallowing transitions which are `eos3a` → other things, I think this variable should be renamed to cover its new meaning. Perhaps `is_conditional_upgrade_path`?

Reply:
sure!